feat: add documentation for newcomers to bauta.js (#64)
axa-group/bauta.js: README.md

- Built-in logging system.
- Written in TypeScript.

## Bauta.js and developer experience

> If you want an overview of what bauta.js is about from a developer experience point of view it may be interesting to check [this section](./docs/developer-experience/readme.md).

Reviewer: ```suggestion
If you want an overview of what bauta.js is about from a developer experience point of view, check the [developer experience section](./docs/developer-experience/readme.md).
```
